src/table/components/HeadCellMenu.tsx

```diff
@@ -87,12 +99,21 @@ export default function HeadCellMenu({
                   className="sn-table-head-menu"
                   aria-labelledby="sn-table-head-menu-button"
                 >
-                  <ListItems itemTitle={translator.get('SNTable.SortItem.SortAscending')}>
+                  <MenuItem
+                    itemTitle={translator.get('SNTable.SortItem.SortAscending')}
```

Comment:
I realized this is the not the name of the string, it is actually `SNTable.MenuItem.SortAscending`